### PR TITLE
Added Blast

### PR DESCRIPTION
**Pull request by mihaic01**
Included Blast in the list of products.

### CODE REVIEW

File: src/content/enterprise/index.md [hunk 1]
```diff
@@ -64,6 +65,7 @@ Some collaborative efforts to make Ethereum enterprise friendly have been made b
 - [Unibright](https://unibright.io/) _a team of blockchain specialists, architects, developers and consultants with 20+ years of experience in business processes and integration_
 - [Zero Services GmbH](https://www.zeroservices.eu/) _provider of managed services spread across co-locations in Europe and Asia. Operates & monitors your nodes securely and reliably_
 
+
```
Comment (minimalsm) [inline src/content/enterprise/index.md:68]:
```suggestion
```

File: src/content/enterprise/index.md [hunk 2]
```diff
@@ -52,6 +52,7 @@ Some collaborative efforts to make Ethereum enterprise friendly have been made b
 ### Products and services {#products-and-services}
 
 - [Alchemy](https://www.alchemy.com/) _provides API services and tools for building and monitoring applications on Ethereum_
+- [Blast](https://blastapi.io/) _a multichain, subscription-based, API platform, featuring the fastest and most reliable blockchain access to Ethereum Archive Mainnet and Tetnets._
```
Comment (minimalsm) [inline src/content/enterprise/index.md:55]: > featuring the fastest and most reliable blockchain

Can we remove the salesy language here, please? We have no way of proving this claim.

> a multichain,

Don't think this is necessarily appropriate information for people interested in enterprise-level **Ethereum**

See other examples for more appropriate language choices here please :)

Comment (mihaic01) [reply src/content/enterprise/index.md]: I changed the text.

Comment (mihaic01) [reply src/content/enterprise/index.md]: Hey @minimalsm ,

Is there something else needed from us for the PR? We can also schedule a call for a demo regarding our Blast platform, if needed.

Comment (mihaic01) [reply src/content/enterprise/index.md]: Hey @minimalsm ,

Happy new year!

I just wanted to know if there is something else needed from us for the PR.

**Review by corwintines — APPROVED** [review]

**Pull request merged**